python/social_bot/tasks.py

```diff
@@ -156,21 +160,29 @@ def __init__(self,
         self._use_curriculum_training = use_curriculum_training
         self._start_range = start_range
         self._is_full_range_in_curriculum = False
+        self._random_goal = random_goal
+        self._distraction_list = distraction_list
+        self._object_list = distraction_list
+        if not goal_name in distraction_list:
+            self._object_list.append(goal_name)
+        self._goals = self._object_list
```

Comment:
Let's use distraction_list + [goal_name] as the range of goals if random_goal is enabled for now. 
If specifying the list for goals is needed, we can define goal_name as a list.